### DISCUSSION TITLE
How do I apply both a spring animation and a static rotate (CSS) on the same item?

### DISCUSSION THREAD

**Discussion by g1sbi** [pmndrs/react-spring#1972] [Support]
I have items that have a static `transform: rotate` applied to them in CSS, and then with a spring animation I translate them on the y axis. All CSS properties are still applied correctly, except for the rotate. Should it work out of the box and I did something wrong?

**Comment by joshuaellis** [comment]
Can you share a codesandbox of what you're trying to achieve?
